docs/src/main/sphinx/connector/pulsar.rst

```diff
@@ -0,0 +1,270 @@
+=================
+Pulsar connector
+=================
+`Apache Pulsar <https://pulsar.apache.org/en/>`_ is a cloud-native, distributed messaging and streaming platform.
+
+This connector allows the use of Pulsar topics as tables in Trino, such that each Pulsar message
+in a Pulsar topic is presented as a row in Trino. And Pulsar topic schema is mapped to Trino
+table schema.
+
+Topics in Pulsar are stored as segments in `Apache BookKeeper <https://bookkeeper.apache.org/>`_. Each topic segment is replicated to some BookKeeper nodes,
+which enables concurrent reads and high read throughput. You can configure the number of BookKeeper nodes,
+and the default number is 3. In Trino Pulsar connector, data is read directly from BookKeeper,
+so Trino workers can read concurrently from horizontally scalable number BookKeeper nodes.
+
+This connector is **read-only** for now. It can only read data from Pulsar topics,
```

Comment:
This whole section should go into a "SQL support" section further down the file. Something like

```
SQL support
-----------

The connector provides **read-only access** to data and metadata in Pulsar topics
with the following statements:

* :doc:`/sql/select`
* :doc:`/sql/show-catalogs`
* :doc:`/sql/show-columns`
* :doc:`/sql/show-schemas`
* :doc:`/sql/show-tables`
* :doc:`/sql/describe`

Other supported commands:

* :doc:`/sql/explain`
* :doc:`/sql/explain-analyze`
* :doc:`/sql/set-session`
* :doc:`/sql/show-functions`
* :doc:`/sql/show-session`
* :doc:`/sql/use`
* :doc:`/sql/values`

No other statements are supported.
```

Reply:
You might have to adapt what is true for this connector and add this section after the "Configuration" section

Reply:
@mosabua thanks for reminding, we will test and add docs.